jkq/ddsim/circuitsimulator.py

```diff
@@ -0,0 +1,113 @@
+"""Backend for DDSIM."""
+
+import logging
+import time
+import uuid
+
+from qiskit.providers import BackendV1, Options
+from qiskit import qobj, QiskitError
+from qiskit.providers.models import BackendConfiguration, BackendStatus
+from qiskit.result import Result
+
+from .jkqjob import JKQJob
+from .jkqerror import JKQSimulatorError
+
+from jkq.ddsim import pyddsim
+
+logger = logging.getLogger(__name__)
+
+
+class CircuitSimulator(BackendV1):
```

Comment:
It is a bit confusing that this class is called `CircuitSimulator` while the class from the Python bindings is named exactly the same.